aws/README.md

```diff
@@ -1,3 +1,93 @@
+# Contributor process
+
+If you have a pull request that contains tests that need to be enabled in CI you can get the ball rolling by
```

Comment:
> If you have a pull request for [Ansible](https://github.com/ansible/ansible/) which contains tests that need to be enabled in CI you can get the ball rolling by
submitting a pull request to [this repository](https://github.com/matt-telstra/aws-terminator/). Start by updating the appropriate test policy in the policy directory (policy
groups are defined below).